codalab/worker_manager/azure_batch_worker_manager.py

```diff
@@ -133,6 +137,9 @@ def start_worker_job(self) -> None:
             task_container_run_options.append(
                 '--env CODALAB_SENTRY_INGEST_URL=%s' % CODALAB_SENTRY_INGEST
             )
+            task_container_run_options.append(
+                '--env CODALAB_SENTRY_ENVIRONMENT=%s' % CODALAB_SENTRY_ENVIRONMENT
```

Comment:
@epicfaace do you think we need this here? Is the sentry sdk initialized in the worker managers?

Reply:
I'm not sure, but we should probably make an issue if that's not the case (I believe that nevertheless, these env variables will probably be passed on to the workers, which **do** have the sentry sdk initialized).